src/assets/toolkit/styles/utils/display.css

```diff
@@ -60,4 +68,16 @@
   .u-lg-block {
     display: block !important;
   }
+
+  .u-lg-inlineBlock {
+    display: inline-block !important;
+  }
+
+  .u-lg-unhideVisually {
```

Comment:
Any reason not to call it `showVisually` instead of `unhide`?

Also, you might dig [the `initial` keyword](http://cssnext.io/features/#initial-value) for this.